src/OrbitGl/GraphTrack.cpp

```diff
@@ -107,7 +108,7 @@ void GraphTrack<Dimension>::DoUpdatePrimitives(PrimitiveAssembler& primitive_ass
 template <size_t Dimension>
 Color GraphTrack<Dimension>::GetColor(size_t index) const {
   if (series_colors_.has_value()) return series_colors_.value()[index];
-  return TimeGraph::GetColor(index);
+  return orbit_gl::GetThreadColor(index);
```

Comment:
This is wrong. This doesn't have any relationship with Threads. The method should be `GetTrackColor` or `GetColor`. You might need to add similar methods non-thread related as they were in TimeGraph

Reply:
Since this is unrelated to GetThreadColor I wouldn't touch it here at all.

Reply:
I left a comment above, also related to GetColor methods that takes a string.